### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by oufmilo**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by oufmilo** [comment]
!testme

**Comment by oufmilo** [comment]
testme

**Comment by oufmilo** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #=================================================
 # COMMON VARIABLES
 #=================================================
-pkg_dependencies="php-geoip php7.0-cli"
+YNH_PHP_VERSION="7.3"
+extra_php_dependencies="php7.3-geoip php7.3-cli"
```
Comment (ericgaspar) [inline scripts/_common.sh:5]:
```suggestion
extra_php_dependencies="php${YNH_PHP_VERSION}-geoip php${YNH_PHP_VERSION}-cli"
```

File: scripts/backup [hunk 2]
```diff
@@ -38,40 +38,40 @@ db_name=$(ynh_app_setting_get $app db_name)
 #=================================================
 # BACKUP THE APP MAIN DIR
 #=================================================
-ynh_script_progression --message="Backing up the main app directory..."
+ynh_print_info --message="Backing up the main app directory..."
```
Comment (ericgaspar) [inline scripts/backup:41]:
```suggestion

```

File: scripts/backup [hunk 3]
```diff
@@ -38,40 +38,40 @@ db_name=$(ynh_app_setting_get $app db_name)
...
 
 ynh_backup --src_path="$final_path"
 
 #=================================================
 # BACKUP THE NGINX CONFIGURATION
 #=================================================
-ynh_script_progression --message="Backing up nginx web server configuration..."
+ynh_print_info --message="Backing up nginx web server configuration..."
```
Comment (ericgaspar) [inline scripts/backup:48]:
```suggestion

```

File: scripts/backup [hunk 4]
```diff
@@ -38,40 +38,40 @@ db_name=$(ynh_app_setting_get $app db_name)
...
-ynh_script_progression --message="Backing up the MySQL database..."
+ynh_print_info --message="Backing up the MySQL database..."
 
 ynh_mysql_dump_db "$db_name" > db.sql
 
 #=================================================
 # BACKUP THE CRON FILE
 #=================================================
-ynh_script_progression --message="Backing up cron configuration..."
+ynh_print_info --message="Backing up cron configuration..."
```
Comment (ericgaspar) [inline scripts/backup:62]:
```suggestion

```

File: scripts/backup [hunk 5]
```diff
@@ -38,40 +38,40 @@ db_name=$(ynh_app_setting_get $app db_name)
...
 
 ynh_backup --src_path="/etc/cron.d/$app"
 
 #=================================================
 # BACKUP postfix conf
 #=================================================
-ynh_script_progression --message="Backing up postfix conf configuration..."
+ynh_print_info --message="Backing up postfix conf configuration..."
```
Comment (ericgaspar) [inline scripts/backup:69]:
```suggestion

```

File: scripts/install [hunk 6]
```diff
@@ -156,4 +163,9 @@ fi
 ynh_systemd_action --service_name=nginx --action=reload
 ynh_systemd_action --service_name=postfix --action=reload
 service php* reload
```
Comment (ericgaspar) [inline scripts/install:165]:
```suggestion

```

File: scripts/upgrade [hunk 7]
```diff
@@ -39,13 +38,13 @@ upgrade_type=$(ynh_check_app_version_changed)
...
-	ynh_app_setting_set --app=$app --key=is_public --value=0
-	is_public=0
-fi
+#if [ "$is_public" = "Yes" ]; then
+#	ynh_app_setting_set --app=$app --key=is_public --value=1
+#	is_public=1
+#elif [ "$is_public" = "No" ]; then
+#	ynh_app_setting_set --app=$app --key=is_public --value=0
+#	is_public=0
+#fi
```
Comment (ericgaspar) [inline scripts/upgrade:47]:
```suggestion

```

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: scripts/_common.sh [hunk 8]
```diff
@@ -1,6 +1,7 @@
 #=================================================
 # COMMON VARIABLES
 #=================================================
-pkg_dependencies="php-geoip php7.0-cli"
+YNH_PHP_VERSION="7.3"
+extra_php_dependencies="php${YNH_PHP_VERSION}-geoip php${YNH_PHP_VERSION}-cli"
```
Comment (ericgaspar) [inline scripts/_common.sh:5]:
```suggestion
extra_php_dependencies="php-geoip php${YNH_PHP_VERSION}-cli"
```

**Review by ericgaspar — COMMENTED** [review]
I am not sure that the package is called with PHP7.3 version...
cf. https://packages.debian.org/fr/buster/php-geoip

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**